docs/reference/content/whats-new.md

```diff
@@ -7,6 +7,27 @@ title = "What's New"
   pre = "<i class='fa fa-level-up'></i>"
 +++
 
+## What's new in 3.12
+
+This release fully supports all MongoDB releases from versions 2.6 to 4.2. Key new features of the 3.12 Java driver release include:
+
+### Security improvements
+
+Client-side field level encryption is supported. Automatic encryption and decryption is available for users of
+[MongoDB Enterprise Advanced](https://www.mongodb.com/products/mongodb-enterprise-advanced), while explicit encryption and decryption is
+available for users of MongoDB Community.
+
+See [Client-side Encryption]({{<ref "driver/tutorials/client-side-encryption.md">}}) for further details.
+
+### Improved interoperability when using the native UUID type
```

Comment:
We don't really have a place in the reference documentation that makes sense to go into detail on this.  It doesn't seem to warrant its own page, but there is no other page that goes into any detail on the various settings.